tekton/tasks/binary-container-build.yaml

```diff
@@ -28,6 +28,9 @@ spec:
     - name: ws-reactor-config-map
     - name: ws-remote-host-auth
 
+  results:
+    - name: task_result
```

Comment:
Same here - but I guess this one might be needed, otherwise we don't get the thing we use to check the error message

Which remind me, could you explain -- in the commit message or in a comment in the YAML -- how this gets us the error message?  Cause it's not through task results but something else, right?

Reply:
this is required for sure, because without results it won't set error msg, I can rename it to task_error or something tho